src/Our.Umbraco.Ditto/ComponentModel/Attributes/DittoProcessorAttribute.cs

```diff
@@ -69,6 +71,56 @@ protected DittoProcessorAttribute()
         internal Type ContextType { get; set; }
 
         /// <summary>
+        /// Returns the MembershipHelper instance
+        /// </summary>
+        protected MembershipHelper Members
+        {
+            get { return new MembershipHelper(Umbraco); }
```

Comment:
You could keep these accessors, but maybe the main exposure point is the ProcessorContext, so you could do Context.Members or  just Members (which internally just calls Context.Members). I dunno if this is a good or bad idea, so open to discussion? @leekelleher?